src/client/components/quickReference/QuickReference.tsx

```diff
@@ -482,13 +482,13 @@ function TextEntryString({
               />
             );
           }
-          case "scaledice":
-          case "scaledamage":
+          case "scaleDice":
+          case "scaleDamage":
```

Comment:
These should not be changed, since they come from an external data source.

Reply:
ah, turns out I had also changed some names in my `node_modules`. I think it should be fixed now.